pkg/doc/did/doc.go

```diff
@@ -670,14 +670,20 @@ func stringArray(entry interface{}) []string {
 
 // JSONBytes converts document to json bytes
 func (doc *Doc) JSONBytes() ([]byte, error) {
+	context := Context
+
+	if len(doc.Context) > 0 {
+		context = doc.Context[0]
```

Comment:
I don’t understand this change. Why are we removing contexts.

Reply:
I didn't remove context just use v1 if context is empty